settings.go

```diff
@@ -65,11 +65,18 @@ func (l *List) compile() (*list, error) {
 		if err != nil {
 			errs = append(errs, err)
 		}
+		sort.Strings(l.Allow)
```

Comment:
This modifies the input slice which can be unexpected behavior to the consumer (if there are any). That is why I had it copied to an internal slice before sort.